dataeng/resources/vault-config.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+
+# make a directory to store vault token helper and config file
+mkdir ${WORKSPACE}/vault-config
```

Comment:
Is there any risk that a job could re-use a workspace that already has a vault-config directory? What if a previous job failed before it performed a clean up? I can't remember if any of our jobs re-use work spaces.

Reply:
Just for clarification each build is a separate job ?

Reply:
In Jenkins terms, each time you run a job, that is called a "build". Each build gets it's own unique BUILD_ID and namespace (separate environment variables/parameters). Depending on how a job is set up, it can either have its own isolated workspace or share one. That's what I don't remember in this case.

Reply:
I have checked from server there are multiple workspaces created for each worker such as 
analytics-exporter-worker@3
analytics-exporter-worker@4
analytics-exporter-worker@5
analytics-exporter-worker@6
after this segregation is there still any chance that workspace could be reused ??